contracts/ETO/ETOCommitment.sol

```diff
@@ -690,44 +696,46 @@ contract ETOCommitment is
         // so if we sold MAX_AVAILABLE_TOKENS that were computed via calculateAmountWithoutFee from MAXIMUM_NUMBER_OF_TOKENS
         // the reverse operation, that is MAX_AVAILABLE_TOKENS + calculatePlatformTokenFee will not always be MAXIMUM_NUMBER_OF_TOKENS
         // while it's probably possible to detect it we put this check here
-        uint256 tokenParticipationFeeInt;
-        if (_totalTokensInt == MAX_AVAILABLE_TOKENS) {
-            uint256 maxTokens = tokenTerms.MAX_NUMBER_OF_TOKENS();
+        uint256 tokenParticipationFee;
+        uint256 maxTokens = tokenTerms.MAX_NUMBER_OF_TOKENS();
```

Comment:
this line has moved from inside to outside the if() condition, not necessary i think

Reply:
true, I had one invariant tested below which I dropped, now indeed it can go back